src/Functions/S2CapContains.cpp

```diff
@@ -0,0 +1,121 @@
+#include "config_functions.h"
+
+#include <Columns/ColumnsNumber.h>
+#include <Columns/ColumnTuple.h>
+#include <DataTypes/DataTypesNumber.h>
+#include <DataTypes/DataTypeTuple.h>
+#include <Functions/FunctionFactory.h>
+#include <Functions/IFunctionImpl.h>
+#include <Common/typeid_cast.h>
+#include <ext/range.h>
+
+#include "../contrib/s2geometry/src/s2/s2latlng.h"
```

Comment:
Similar to other thirdparty libraries, this needs to be a relative import, example:  `s2/s2latlng.h` and not an absolute import using the path `../contrib/...`